rust/connlib/snownet/src/allocation.rs

```diff
@@ -766,7 +767,7 @@ impl ChannelBindings {
         let channel = self.inner.get_mut(&channel_number)?;
         channel.record_received(now);
 
-        debug_assert!(channel.bound); // TODO: Should we "force-set" this? We seem to be getting traffic on this channel ..
+        // debug_assert!(channel.bound); // TODO: Should we "force-set" this? We seem to be getting traffic on this channel ..
```

Comment:
@thomaseizinger this was failing when I was testing but everything worked anyways when commenting it out

Reply:
I'll look into this. From memory it is a race condition of our local state that I thought could never happen.

Reply:
This is interesting. If this `debug_assert` fails then it means we are receiving traffic on this channel but the relay has not yet given us a success response for the binding. Can you share logs from when this got triggered? It seems very unlikely that the relay would not send us a success response but relay data to us.